client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -130,7 +129,7 @@ class Upload extends Component {
 	}
 
 	redirectToWpAdmin() {
-		page( `https://${ this.props.siteSlug }/wp-admin/theme-install.php` );
+		window.location = `${ this.props.siteAdminUrl }theme-install.php`;
```

Comment:
Thanks for the fix 👍 

Another location that could benefit from a theme install URL selector FWIW.